CMakeLists.txt

```diff
@@ -90,6 +90,63 @@ list(APPEND rcutils_sources
   include/rcutils/logging_macros.h)
 include_directories("${CMAKE_CURRENT_BINARY_DIR}/include")
 
+###############################################################################
+# Create version file in build/<package>/include/<package> folder which is already in the include list
+set(TMP_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/include/${PROJECT_NAME})
+set(RCUTILS_VERSION_FILE_NAME ${TMP_INCLUDE_DIR}/version.h)
+set(NEED_TO_CREATE_VERSION_FILE false)
```

Comment:
Nitpick: the CMake constant is `FALSE`.

Same below for `TRUE`.

Reply:
fixed